src/server/transaction.h

```diff
@@ -597,6 +596,7 @@ class Transaction {
   // We need values as well since we reorder keys, and we need to know what value corresponds
   // to what key.
   absl::InlinedVector<std::string_view, 4> kv_args_;
+  absl::InlinedVector<uint64_t, 4> kv_fp_;
```

Comment:
please add a comment what this is, you even don't use the fingerprint alias you introduced in common 🙂